### PR TITLE
[Webportal] Add user logout and registration

### PR DESCRIPTION
**Pull request by abuccts**
Add user logout and registration.

### CODE REVIEW

File: webportal/src/app/user/user-login/user-login-nav.component.ejs [hunk 1]
```diff
@@ -1,9 +1,13 @@
 <% if (cookies.get("token")) { %>
-  <li class="dropdown user user-menu">
-    <a href="#" data-toggle="control-sidebar">
+  <li class="dropdown">
+    <a href="#" class="dropdown-toggle" data-toggle="dropdown">
       <!-- img src="" class="user-image" alt="User Image" -->
       <span class="hidden-xs">Welcome, <%= cookies.get("user") %></span>
     </a>
+    <ul class="dropdown-menu">
+      <li><a href="#" onclick="window.location.replace('/register.html')">New user</a></li>
```
Comment (hwuu) [inline webportal/src/app/user/user-login/user-login-nav.component.ejs]: Will the 'new user' link still be available even if the current user is not an admin? Adding/removing users is a separate functionality (management operations of admins only) other than login/logout and registration (self-serving operations available for all users). In addition, the top-right menu usually contains the operations closely related to the current user only (see github's menu). I suggest we put this user management feature in a separate place.

File: webportal/src/app/user/user-register/user-register.component.ejs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<section class="content-header"><%= breadcrumb({ breadcrumbTitle: "User Register" }) %></section>
```
Comment (hwuu) [inline webportal/src/app/user/user-register/user-register.component.ejs]: Register --> Registration

Comment (abuccts) [reply webportal/src/app/user/user-register/user-register.component.ejs]: only change this place?

File: webportal/src/app/user/user-login/user-login-nav.component.ejs (hunk 1 above)
Comment (abuccts) [reply webportal/src/app/user/user-login/user-login-nav.component.ejs]: need to support in authentication first

File: webportal/src/app/user/user-register/user-register.component.ejs (hunk 2 above)
Comment (hwuu) [reply webportal/src/app/user/user-register/user-register.component.ejs]: Yes please.

File: webportal/src/app/user/user-login/user-login-nav.component.ejs (hunk 1 above)
Comment (hwuu) [reply webportal/src/app/user/user-login/user-login-nav.component.ejs]: Let's do it in this way for now and then improve in the future.

File: webportal/src/app/user/user-register/user-register.component.ejs (hunk 2 above)
Comment (abuccts) [reply webportal/src/app/user/user-register/user-register.component.ejs]: updated

**Pull request merged**